styles.css

```diff
@@ -2,6 +2,16 @@
   font-family: 'Open Sans', sans-serif;
 }
 
+/* BODY & <h> Tag Styling */
+
+body {
+  background-color: rgb(133,216,223)
+  font-family: Arial;
+  background: var(#222222);
+  margin: 0;
```

Comment:
At the top level you shouldd have 

* { padding: 0; margin: 0;}

and then remove line 11